src/components/ui/streamer.tsx

```
@@ -0,0 +1,154 @@
+import ReactMarkdown from "react-markdown";
+import { Button } from "@/components/ui/button";
+import { Textarea } from "@/components/ui/textarea";
+import { Pencil, Save } from "lucide-react";
+
+interface StreamerProps {
+  pdfUrl: string;
+  editedPdfUrl: string | null;
+  setEditedPdfUrl: (url: string) => void;
+}
+
+const Streamer: React.FC<StreamerProps> = ({ pdfUrl, editedPdfUrl, setEditedPdfUrl }) => {
```

ESLint

[error] 'editedPdfUrl' is defined but never used.

(@typescript-eslint/no-unused-vars)